cpskin/workflow/permissions.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+from AccessControl.SecurityInfo import ModuleSecurityInfo
+from Products.CMFCore.permissions import setDefaultRoles
+
+
+security = ModuleSecurityInfo('cpskin.locales')
```

Comment:
Should be 'cpskin.workflow' instead of locales, isn't it ?

Reply:
Indeed